bigtable/scheduled-backups/README.md

```diff
@@ -0,0 +1,127 @@
+# Scheduled Backups
+
+This example shows how to use Cloud Scheduler and Cloud Functions to configure a
+schedule that creates Cloud Bigtable backups periodically.
+
+The idea is to have a Cloud Scheduler job that invokes a Cloud function by
+sending a message to the Pub/Sub topic which contains information about the
+Cloud Bigtable backup creation request. Then the Cloud function initiates a
+backup using Cloud Bigtable Java API.
+
+## Create scheduled backups
+
+1.  Clone this directory and make changes to
+    `./config/scheduled-backups.properties` file to match your configuration.
+    Fields that need to be updated start with `"replace"`.
+
+2.  Create a Cloud Pub/Sub topic `cloud-bigtable-scheduled-backups` that serves
+    as the target of the Cloud Scheduler job and triggers the Cloud function.
+    For example:
+
+```
+gcloud pubsub topics create cloud-bigtable-scheduled-backups --project <project-id>
+```
+
+3.  Create and deploy a Cloud Function `cbt-create-backup-function` which is
+    called whenever a Pub/Sub message arrives in
+    `cloud-bigtable-scheduled-backups` topic:
+
+```
+./scripts/scheduled_backups.sh deploy-backup-function
+```
+
+4.  Deploy the scheduled backup configuration to Cloud Scheduler:
+
+```
+./scripts/scheduled_backups.sh create-schedule
+```
+
+## Email notification of backup failures
+
+To get email notifications on backup creation failures, follow these steps:
+
+1.  Follow this
+    [guide](https://cloud.google.com/monitoring/support/notification-options#email)
+    to add your email address as a notification channel.
+
+2.  Create and deploy a custom metrics configuration file to filter logs
+    generated by Cloud Functions, Cloud Scheduler and Cloud Bigtable. We use
+    [Deployment Manager](https://cloud.google.com/deployment-manager/docs/quickstart)
+    to create custom metrics. The example file can be found in
+    `./config/metrics.yaml`. Deploy the custom metrics in Cloud Logging:
+
+```
+./scripts/scheduled_backups.sh add-metrics
+```
+
+After this, you should see two user-defined metrics under `Logs-based Metrics`
+in Cloud Logging.
+
+1.  Go to `Logs-based Metrics` in Cloud Logging and select `Create alert from
+    metric` option for each of the two metrics created in the above step. From
+    there, you can choose `Aggregrator`, such as `sum` or `mean`, for the target
+    metric, and define what the condition of triggering an alert is, e.g., any
+    time series violates that the value is above 0 for 1 minute.
+
+2.  Add notification channels you just created to alerting policies.
+
+### APIs and IAM roles setup
+
+The diagram below focuses on the actions flow between human roles and APIs.
+<img src="https://drive.google.com/uc?export=view&id=1YHUh5FKSuNMTSj6_E7Ehsq31RHNPu2Wu" width="600" height="auto" />
+
```

Comment:
Are you sure you want to serve images from drive rather than checking them in here.  (I think it's better to check in)

Reply:
Thanks! Moved it under /images.

Reply:
Yet somehow the README didn't change.

Reply:
Maybe you are looking at an outdated file? It is [updated](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/4590/files#diff-d10da0641c4fc5a20fc6d436476f8b9de7785f740931dc5c04cbff2d57fb3638R23)